recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.13.1" %}
+{% set version = "0.14.0" %}
```

Comment:
Technically we can now loosen the Python requirement. py35 is now tested. :smile: